packages/ddp-server/livedata_server_tests.js

```diff
@@ -178,6 +187,75 @@ Tinytest.addAsync(
   }
 );
 
+Tinytest.addAsync(
+  "livedata server - connection in method called from publish function",
+  function (test, onComplete) {
+    makeTestConnection(
+      test,
+      function (clientConn, serverConn) {
+        onSubscription[serverConn.id] = function (id) {
+          delete onSubscription[serverConn.id];
+          test.equal(id, serverConn.id);
+          clientConn.disconnect();
+          onComplete();
+        };
+        clientConn.subscribe("livedata_server_test_sub_method", serverConn.id);
+      }
+    );
+  }
+);
+
+var methodCallResults = {};
+
+Meteor.publish("livedata_server_test_sub_with_method", function (connectionId) {
+  if (! methodCallResults[connectionId]) {
+    methodCallResults[connectionId] = [];
+  }
+  methodCallResults[connectionId].push(Meteor.call('livedata_server_test_inner'));
```

Comment:
Here `DDP._CurrentInvocation.get()` shoud be pushed to `methodCallResults` now, no?